stdlib/number.gr

```diff
@@ -21,27 +21,31 @@ import { parseInt } from "runtime/stringUtils"
 import { newFloat64, newInt64 } from "runtime/dataStructures"
 import Tags from "runtime/unsafe/tags"
 
+// Constants
+let infinity = 1.0 / 0.0
+let nan = 0.0 / 0.0
```

Comment:
Should we export these and add docs?

Reply:
They are exported in #1443 my plan was once that is merged to rebase this commit and remove those

Reply:
Why wouldn't you just include it *here* and remove it fro #1443 ? You grew that PR so large that it'll take more time to go through it.